tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -690,6 +693,8 @@ task formatPipelineOutputs {
     outputs_dict["picard_quality_distribution_pdf_file"]="~{picard_quality_distribution_pdf}"
     outputs_dict["fp_summary_metrics_file"]="~{picard_fingerprint_summary_metrics}"
     outputs_dict["fp_detail_metrics_file"]="~{picard_fingerprint_detail_metrics}"
+    outputs_dict["contamination"]=str(~{contamination})
```

Comment:
@mmorgantaylor I was hoping to just use sed within the CalculateContamination task to truncate the digits, but the problem with that approach is sometimes the number is in scientific notation e.g. 0.001 vs 1E-3. It looks like wdl's read_float() automatically converts scientific notation to decimals. So this seems like a good place to truncate the digits. Do you mind adding it to your PR? (I don't have an easy way to test my the warp wdl…)

I think either

`'%.5f'%(float(~{contamination}))`

or 

`'%.5f'%(~{contamination})`

should work, depending on the type of `~{contamination}` in python. Or if you have a different approach please use that, since it seems like you have something working in your branch.

Reply:
ah yeah sure thing, will add that and test it now!

Reply:
Thanks!